src/main/java/org/isf/utils/jobjects/LocalDateSupportingJDateChooser.java

```diff
@@ -0,0 +1,57 @@
+package org.isf.utils.jobjects;
```

Comment:
File is missing license header; it can be automatically added by running `mvn license:format` in the root directory of the repo.